projects/SCION/default.nix

```diff
@@ -0,0 +1,48 @@
+{
+  lib,
+  pkgs,
+  sources,
+}@args:
+{
+  # NOTE:
+  # - Check `projects/models.nix` for a more detailed project structure
+  # - Each program/service must have at least one example
+  # - Set attributes to `null` to indicate that they're needed, but not available
+  metadata = {
+    summary = "";
+    subgrants = [
+      "FooBar"
+      "FooBar-cli"
```

Comment:
Search for the project in the [NLnet projects page](https://nlnet.nl/project/) and include the URL handle for the things related to SCION (see [triaging template](https://github.com/ngi-nix/ngipkgs/blob/main/.github/ISSUE_TEMPLATE/project-triaging.yaml) for an example).